pkg/dogstatsd/packets/buffer.go

```diff
@@ -8,6 +8,15 @@ package packets
 import (
 	"sync"
 	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/telemetry"
+)
+
+var (
+	tlmBufferFlushedTimer = telemetry.NewCounter("dogstatsd", "packets_buffer_flush_timer",
+		nil, "Count of packets buffer flush triggered by the timer")
+	tlmBufferFlushedFull = telemetry.NewCounter("dogstatsd", "packets_buffer_flush_full",
+		nil, "Count of packets buffer flush triggered because the buffer is full")
```

Comment:
Could we move the telemetry to `pkg/dogstatsd/packets/telemetry.go` ?

Reply:
Sure, if you feel strongly about it, I can move it back.